app/create-pr.js

```diff
@@ -0,0 +1,171 @@
+/** source for cloud func, not auto-deployed, copy/paste */
```

Comment:
Please review this file.

You'll need to set up your own Google cloud function with minimal specs, then paste this code into it.